lib/asciidoctor-epub3/packager.rb

```diff
@@ -203,21 +203,50 @@ def add_cover_page doc, spine_builder, manifest
     nil
   end
 
-  def add_images_from_front_matter
-    (::File.read 'front-matter.html').scan ImgSrcScanRx do
-      resources do
-        file $1
+  def get_frontmatter_files doc
+    if doc.attr? 'epub3-frontmatterdir'
+      fmdir = doc.attr 'epub3-frontmatterdir'
+      fmglob = 'front-matter.*\.html'
+      unless Dir.exist? fmdir
```

Comment:
`fmdir` needs to be resolved relative to `docdir` and not to current working directory. See #301